python/ray/dag/p2p_node.py

```diff
@@ -26,11 +23,10 @@ class _P2POperation(_NcclOperation):
 
     def __init__(self):
         super().__init__()
-        # [CL] The input reader and output writer for the P2P operation.
-        self.input_reader: Optional[ReaderInterface] = None
-        self.output_writer: Optional[WriterInterface] = None
```

Comment:
We should change `input_reader` into `input_channel`, and `output_writer` into `output_channel`, thus skipping the passed `ch` in `execute`. Also, are they the same channel or different channels?